### PR TITLE
adding elasticache tags

### PR DESCRIPTION
**Pull request by jstenhouse**
@stevenjack

I wasn't able to test this locally because aruba was giving me some issues. 

I *think* this should work.

### CODE REVIEW

**Comment by jstenhouse** [comment]
Build failed because Rubocop doesn't like an implementation of `method_missing`. Looks unrelated to my changes.

**Comment by jstenhouse** [comment]
Bump :)

File: lib/cfndsl/aws/types.yaml [hunk 1]
```diff
@@ -1074,6 +1075,9 @@ Types:
  EC2Tag:
   Key: String
   Value: String
+ ElastiCacheTag:
```
Comment (gergnz) [inline lib/cfndsl/aws/types.yaml]: Any reason we can't use "EC2Tag" type?

Comment (kornypoet) [inline lib/cfndsl/aws/types.yaml]: We definitely shouldn't be creating new named tags. AWS has taken to calling them generically resource tags now: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html
We use EC2Tag in most places. I vote for keeping that pattern for the moment.

Comment (jstenhouse) [inline lib/cfndsl/aws/types.yaml]: Nope, EC2Tag works fine. I saw HostedZoneTag, Route53HealthCheckTags, etc and thought to follow that pattern, but I think it makes more sense to always use EC2Tag or rename them all to just Tag.

**Comment by kornypoet** [comment]
👍

**Comment by gergnz** [comment]
thanks @kornypoet and @jstenhouse

**Pull request merged**